### PR TITLE
Nick  accessibility fixes 8 2020

### PR DESCRIPTION
**Pull request by nicholasnadel**
See https://docs.google.com/spreadsheets/d/1_A9D5Mx0WVnEtOnTNJ3NWEgzFE9aV6qAWlcOLqzB9z4/edit?usp=sharing

### CODE REVIEW

File: app/data_definitions/from_cascade/Shared Fields/image.xml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 <system-data-structure>
-	<group identifier="image" label="Image" multiple="true">
+	<group identifier="imageNICKWASHERE" label="Image" multiple="true">
```
Comment (corejh) [inline app/data_definitions/from_cascade/Shared Fields/image.xml:2]: is this an easter egg or debug artifact?

Comment (nicholasnadel) [reply app/data_definitions/from_cascade/Shared Fields/image.xml:2]: haha! oops. Will fix shortly

Comment (nicholasnadel) [reply app/data_definitions/from_cascade/Shared Fields/image.xml]: Fixed below in e03430f

**Review by corejh — CHANGES_REQUESTED** [review]
Looks good, but can we resolve those conflicts before merging. use whichever are the cleanest versions of those files

**Comment by nicholasnadel** [comment]
> Looks good, but can we resolve those conflicts before merging. use whichever are the cleanest versions of those files

Should be resolved now in 78fafcf

**Review by cbryant24 — APPROVED** [review]

**Pull request merged**